PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -422,25 +432,26 @@ def miniAOD_customizeCommon(process):
     task.add(process.patJetPuppiCharge)
 
     noDeepFlavourDiscriminators = [x.value() for x in process.patJets.discriminatorSources if not "DeepFlavour" in x.value()]
+    if process.patJets.jetSource.value() == "akCs4PFJets": noDeepFlavourDiscriminators = ['pfChargeBJetTags']  #dummy entry
```

Comment:
@slava77   This is just a placeholder to get this working.  Can you advise how to do this properly, based on the era?  The issue is that btagDiscriminators can only contain predefined taggers, and complains when I hand it the taggers with 'akCs4PF' appended at the end.

Reply:
I'd guess `(pp_on_AA_2018 | pp_on_PbPb_run3)..toModify(noDeepFlavourDiscriminators, ['pfChargeBJetTags'])` ; does it not work?

Reply:
That's giving the following python error:
TypeError: 'list' object is not callable

 I also need a pointer for the other instance:  
    if not process.patJets.jetSource.value() == "akCs4PFJets":
        process.patJets.discriminatorSources.extend([
            cms.InputTag('pfDeepCSVDiscriminatorsJetTags:BvsAll' ),
            cms.InputTag('pfDeepCSVDiscriminatorsJetTags:CvsB'   ),
            cms.InputTag('pfDeepCSVDiscriminatorsJetTags:CvsL'   ),
        ])

Thanks!

Reply:
toReplaceWith?

Reply:
Like this?
 (pp_on_AA_2018 | pp_on_PbPb_run3).toReplaceWith(noDeepFlavourDiscriminators,['pfChargeBJetTags'])

That gives: 
    raise TypeError("toReplaceWith does not work with type "+str(type(toObj)))
TypeError: toReplaceWith does not work with type <type 'list'>

Reply:
it may need a VInputTag specified, I guess

Reply:
But bTagDiscriminators is just a list of strings, no?

Reply:
ok, I may need to check this interactively with a release setup, not just guessing.